css/style2.css

```diff
@@ -0,0 +1,93 @@
+/* myStyle.css */
+body {
+    text-align: center;
+    background-color:grey;
+    color: #FFFFFF;
+   }
+   h1 {
+    color:goldenrod;
+    font-size: 26pt;
+    font-weight: bold;
+    text-align: center;
+    font-family: fantasy;
+    line-height: 45pt;
+   
+   }
+
+   h2 {
+    color:white;
+    font-size: 26pt;
+    font-weight: bold;
+    text-align: center;
+    font-family: fantasy;
+    line-height: 45pt;
+   
+   }
+
+   h3 {
+    color:white;
+    font-size: 26pt;
+    font-weight: bold;
+    text-align: center;
+    font-family: fantasy;
+    line-height: 45pt;
+   
+   }
+
+   h4 {
+    color:white;
+    font-size: 26pt;
+    font-weight: bold;
+    text-align: center;
+    font-family: fantasy;
+    line-height: 45pt;
+    
+  
+   
+   }
+
+
+   p {
+   
+   
+    padding: 30px;
+    margin: 50px;
+  }
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  
+  }
+
+  li a {
+    display: block;
+    color:powderblue;
+    padding: 8px 16px;
+    text-decoration: none;
+    float:center; 
+  }
+  
+  /* Change the link color on hover */
+  li a:hover {
+    background-color: rgba(85, 85, 85, 0.767);
+    color: yellow;
+  }
+
+  header {
+    background-color:whitesmoke;
+    color:brown;
+    font-size: 12pt;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 80px;
+    z-index:-1;
```

Comment:
From a design and functionality viewpoint, it's better to put the order of the header as the largest number so it stays on top of everything. That way when a user scrolls there won't be text or images covering the nav bar and they can actually use it whenever they want. Try changing the z-index in both style files to something like 5 for example and see what happens.